tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -98,10 +98,7 @@ function configure_different_shims_base() {
 			fi
 		fi
 
-		cat << EOF | tee "$shim_file"
```

Comment:
Why is this being deleted. You would need to create the "$shim_file" first in order to create a symlink to it in line 101 right? Let me know if I am missing something.

Reply:
I'm linking directly from `/opt/kata/bin/containerd-shim-kata-v2`, as we don't need any content in the file because the configuration path is already set as part of the CRI engine (CRI-O and containerd) configuration,

Reply:
ok. Got it, thanks for the explanation.